main.py

```diff
@@ -64,6 +64,49 @@ def mood(result):
     elif result=="Surprise":
         return 'You seem surprised! Hopefully its some good news. Lets celebrate it with some great music!'
 
+
+def url(result):
```

Comment:
Change  the function name to `provide_url` and make changes everywhere in the function calls since It is overriding the `url` variable when we are providing the URL to fetch the Image.